test/uri/EndpointTest.ooc

```diff
@@ -0,0 +1,44 @@
+use uri
+use collections
+use base
+use unit
+
+EndpointTest: class extends Fixture {
+	init: func {
+		super("Endpoint")
+		this add("parse", func {
+			one := t"one"
+			two := t"two"
+			three := t"three"
+			host := one + t"." + two + t"." + three
+			port := t"1232"
+			host_port := host + t":" + port
+			endpoint := Endpoint parse(host_port take())
+			list := endpoint host
+			expect(list[0] == one)
```

Comment:
I had some problems with `is equal to()` but not on all tests. I changed to `is equal to()` where it worked.